### ISSUE TITLE
🐛 [Bug]: DialogBox secondary pop-up window. After closing the pop-up window, the mask becomes invalid and the buttons on the main page can receive events normally.

### ISSUE THREAD

**Issue by horadrim1015** [opentiny/tiny-vue#495]
### Version

3.10.0

### Vue Version

3.2.31

### Link to minimal reproduction

https://opentiny.design/tiny-vue/playground?cmpId=dialog-box&fileName=secondary-dialog.vue&apiMode=Composition

### Step to reproduce

1、点击“二级弹窗”
2、点击“子弹窗”
3、关闭“子弹窗”
4、再次点击“二级弹窗”

**Comment by zuowendong** [comment]
经过测试，发现在二级菜单关闭之后，一级菜单的外层wrapper元素上会添加一个css属性：pointer-events: none
移除该属性，遮罩层生效，也无法点击主页面元素。
可能是因为同一元素上设置fixed和pointer-events，会让元素的层级失效，也让事件透传失效吧
测试用例代码：
```vue
<template>
  <div class="main">
    <div class="btn" @click="handle">show box dialog</div>
    <div v-if="msg">{{ msg }}</div>
    <div class="mask">
      <div class="box">box</div>
    </div>
  </div>
</template>

<script setup>
import { ref } from "vue";

let msg = ref("");
function handle() {
  msg.value += "I got clicked~";
  console.log("click");
}
</script>

<style>
.btn {
  cursor: pointer;
}
.mask {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: rgba(0, 0, 0, 0.6);
  display: flex;
  align-items: center;
  justify-content: center;

  pointer-events: none;
}
.box {
  width: 200px;
  height: 100px;
  box-shadow: 1px 1px 1px 1px #333;
  background-color: #fff;
}
</style>
```

**Comment by kagol** [comment]
Fixed by @Zuowendong on #503
